### PR TITLE
Adding Error page and routing

### PR DESCRIPTION
**Pull request by AbSamad99**
Added a simple ErrorScreen page for when a user tries to visit an invalid endpoint. Added this route to App.js and edited the Redirect component to redirect to the /404 endpoint

### CODE REVIEW

File: src/screens/ErrorScreen/styles.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import "../../scss/variables";
+@import "../../scss/_base";
+
+#error-screen-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.big-text {
```
Comment (Gajendra-NL) [inline src/screens/ErrorScreen/styles.scss:10]: I think its good to nest styles in scss.
like
#error-screen-div {
  .....
  .big-texx {
      .....
   }
}

Comment (AbSamad99) [reply src/screens/ErrorScreen/styles.scss:10]: Oh I was not familiar with how scss worked so i used the default css syntax

File: src/App.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ import Products from './screens/Products';
 import DailyGrow from './screens/DailyGrow';
 import Consulting from './screens/Dashboard';
 import GreenHouse from './screens/GreenHouse';
+import ErrorScreen from './screens/ErrorScreen';
```
Comment (vyasriday) [inline src/App.js:17]: Just like others, please rename it as Error only instead of ErrorScreen

Comment (AbSamad99) [reply src/App.js:17]: Will do. So I make the changes and make another commit?

Comment (vyasriday) [reply src/App.js:17]: Yes

File: src/screens/Error/styles.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+@import "../../scss/variables";
+@import "../../scss/_base";
+
+#error-screen-div {
```
Comment (nataliepina) [inline src/screens/Error/styles.scss:4]: Just make this is a class instead of an #id we'd like to try to avoid using id unless necessary. :)

Comment (AbSamad99) [reply src/screens/Error/styles.scss:4]: will do!

**Pull request merged**